beeline.go

```diff
@@ -101,8 +101,7 @@ type Config struct {
 }
 
 func IsClassicKey(config Config) bool {
-	// classic key has 32 characters
-	return len(config.WriteKey) == 32
+	return (&libhoney.Config{APIKey: config.WriteKey}).IsClassic()
```

Comment:
This is .. not beautiful, but it also isn't in the critical performance path.

I figure we have a few options here:

1. leave it as it is
1. duplicate the logic from libhoney anyway
1. go back and make a public function in libhoney that is easier to call in this context?
1. something else I haven't thought of 🙃

Reply:
My preference would be for 3, as I think that's closer to the approach we landed on for the other libhoneys where we add a static class method. `libhoney.IsClassicAPIKey(config.WriteKey)` feels like it would be a lot nicer here

Reply:
#3 is implemented here if we're happy with it https://github.com/honeycombio/libhoney-go/pull/239

Reply:
Approved the libhoney-go alternative func. I guess this func can be removed and just use the new libhoney-go IsClassicKey once it's avaialble.

Reply:
I left the local `IsClassicKey` in place as a wrapper around the libhoney function as it is a public function and I didn't want to make an unnecessary breaking change.